### PR TITLE
updates home page to use sg-paginator

### PR DESCRIPTION
**Pull request by pflopez**

### CODE REVIEW

File: src/interface/src/app/standalone/planning-areas/planning-areas.component.html [hunk 1]
```diff
@@ -102,24 +102,13 @@ <h3>Home</h3>
...
-      Page size :
-      <select (change)="changePageSize($event)" [value]="pageOptions.limit">
-        <option value="5">5</option>
-        <option value="12">12</option>
-        <option value="50">50</option>
-      </select>
-    </div>
-  </div>
+  <sg-paginator
+    *ngIf="pages$ | async"
```
Comment (pflopez) [inline src/interface/src/app/standalone/planning-areas/planning-areas.component.html:106]: I added this ngIf bc the paginator has some math with the pageCount on ngInit, and without it it doesn't render them properly

Screen Shot 2024-06-24 at 11 25 09

Comment (lastminutediorama) [reply src/interface/src/app/standalone/planning-areas/planning-areas.component.html:106]: Ok. I thought I fixed that in the paginator, but I'll review again; we shouldn't have to do this.

File: src/interface/src/app/standalone/planning-areas/query-params.service.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export interface QueryParams extends Partial<Sort> {
 
 @Injectable()
 export class QueryParamsService {
-  readonly defaultLimit = 12;
+  readonly defaultLimit = 10;
```
Comment (lastminutediorama) [inline src/interface/src/app/standalone/planning-areas/query-params.service.ts:19]: For the future, I wonder if this should be an imported const or even a user setting (I already forgot that we have a user prefs table) but this is fine for now.

**Review by lastminutediorama — APPROVED** [review]

File: src/interface/src/app/standalone/planning-areas/query-params.service.ts (hunk 2 above)
Comment (pflopez) [reply src/interface/src/app/standalone/planning-areas/query-params.service.ts:19]: an option i was considering is doing something similar to what we are doing with the sort options

```
// Injection token used as default sorting options for this service
export const DEFAULT_SORT_OPTIONS = new InjectionToken<Sort>(
  'DEFAULT_SORT_OPTIONS'
);
```

and let the parent component handle these via injection token. What do you think?

**Pull request merged**